test/podFailover/README.md

```diff
@@ -17,50 +17,37 @@ export REGISTRY=<your-personal-docker-repo>
 make pod-failover-test-containers
 ```
 
-### Create metrics service
+### Run failover test
 
-- Create the metrics service where the workload pod can publish it's metrics using the command mentioned below:
+- Modify test/podFailover/controller/chart/values.yaml
```

Comment:
We can change this in subsequent PR, but we should use `--set` to override values or provide a separate overriding values file through the `--values` parameter rather than recommending modification of the default `values.yaml` file.